LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/parsing/pseudo-elements-valid.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Test pseudo elements parsing of valid selectors</title>
```

Comment:
We should also test that `::view-transition-old(xxx):only-child` parses (which I think this patch doesn't support yet).